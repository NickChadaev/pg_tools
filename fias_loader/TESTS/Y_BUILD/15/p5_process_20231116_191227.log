2023-11-16 19:12:27 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_15 postgres /home/n.chadaev@abrr.local/Y_BUILD/15/stage_gar_c_15.csv 2023-11-13
--------------------------------------------------------
[19:12:27] ... Северная Осетия - Алания Респ, id_region: 16, Execute: /home/n.chadaev@abrr.local/Y_BUILD/15/stage_gar_c_15.csv, process_id:25345
[19:12:27] ... Start process
[19:12:27] ...  -- Version
[19:12:27] ...  -- Очистка от данных
[19:12:28] ...  -- Убираем индексы
[19:12:28] ...  -- Set UNLOGGED
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[19:12:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_REESTR_OBJECTS_20231113_39f1bef6-19d1-4c9a-a94c-437d5599e359.XML
[19:16:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_ADDR_OBJ_20231113_68d70f18-5d22-4c11-bf53-20392786d00d.XML
[19:17:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_HOUSES_20231113_e4ad3517-1fdf-4d54-99a7-2662588bb1ab.XML
[19:21:37] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_STEADS_20231113_f498996d-26a3-4d54-8854-00b5d4bb7244.XML
[19:22:20] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_APARTMENTS_20231113_62c74dc4-f823-42a5-a442-767a1a366168.XML
[19:24:13] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_ROOMS_20231113_701ff29e-f447-469a-948e-6cd0dfcf2b64.XML
[19:24:14] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_CARPLACES_20231113_f3ff08f7-825a-4875-a502-2c2fab89d1aa.XML
[19:24:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_ADM_HIERARCHY_20231113_ce1b3c51-c3bd-486a-bcc6-81d43deeaffd.XML
[19:29:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_MUN_HIERARCHY_20231113_0f3c4a15-c6c0-4397-8aaf-fdb86683e9e4.XML
[19:33:39] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_CHANGE_HISTORY_20231113_32040339-8f38-47de-a91e-22b608605ddc.XML
[19:48:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_ADDR_OBJ_DIVISION_20231113_373787f6-a759-4cbe-83bf-4912bffa0ac9.XML
[19:48:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_ADDR_OBJ_PARAMS_20231113_6e32e928-5288-4db2-be7a-1a84c67f2342.XML
[19:48:38] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_STEADS_PARAMS_20231113_f6d8d2ef-a248-4274-a04e-5c4c7083d5a1.XML
[19:57:37] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_HOUSES_PARAMS_20231113_224f99e8-0f6c-4974-b1ad-7ae7aca18495.XML
[20:25:37] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_APARTMENTS_PARAMS_20231113_3c8223d5-a3ce-4fa9-8541-76f5173c3aed.XML
[20:28:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_ROOMS_PARAMS_20231113_cbbda276-c50c-4833-8030-b09f81ee93d9.XML
[20:28:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_CARPLACES_PARAMS_20231113_b317abcb-5f98-4c59-ba08-856ab60cfbc7.XML
[20:28:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/15/AS_NORMATIVE_DOCS_20231113_6b295d73-3a83-4b4b-8234-352393863e1e.XML
[20:29:35] ...  -- Data finish
[20:29:35] ...  -- Set LOGGED
[20:29:46] ... Stop process
--------------------------------------------------------
