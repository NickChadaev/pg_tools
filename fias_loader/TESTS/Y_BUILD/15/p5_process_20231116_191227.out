 save_gar_version 
------------------
               83
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                83
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 362881
gar_fias.as_addr_obj: 4166
gar_fias.as_houses: 344002
gar_fias.as_steads: 56653
gar_fias.as_apartments: 149776
gar_fias.as_rooms: 1208
gar_fias.as_carplaces: 416
gar_fias.as_adm_hierarchy: 369469
gar_fias.as_mun_hierarchy: 369475
gar_fias.as_change_history: 1209967
gar_fias.as_addr_obj_division: 5
gar_fias.as_addr_obj_params: 51696
gar_fias.as_steads_params: 741785
gar_fias.as_houses_params: 2197106
gar_fias.as_apartments_params: 256799
gar_fias.as_rooms_params: 1175
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 51631
